pkg/scheduler/core/generic_scheduler.go

```diff
@@ -434,21 +441,57 @@ func (g *genericScheduler) prioritizeNodes(
 		return nil, scoreStatus.AsError()
 	}
 
-	if klog.V(10).Enabled() {
-		for plugin, nodeScoreList := range scoresMap {
-			for _, nodeScore := range nodeScoreList {
-				klog.InfoS("Plugin scored node for pod", "pod", klog.KObj(pod), "plugin", plugin, "node", nodeScore.Name, "score", nodeScore.Score)
-			}
-		}
-	}
-
 	// Summarize all scores.
 	result := make(framework.NodeScoreList, 0, len(nodes))
+	totalPluginScores := make(map[string]int64)
 
 	for i := range nodes {
 		result = append(result, framework.NodeScore{Name: nodes[i].Name, Score: 0})
 		for j := range scoresMap {
```

Comment:
please rename j to something more meaningful. I think it refers to plugin here?

Reply:
Come to think that this is not a very efficient implementation of nested loops, for 2 reasons:

1. we can save lookups by doing:
    ```
    for plugin, nodeScores := range scoresMap
    ```
2. Iterating over the map in the outer loop and nodes in the inner loop would keep the slices closer to faster caches in the CPU.

Can you fix this in a separate PR? It would be nice if it shows in any performance benchmarks, but plugins are probably way slower to run. Maybe it will show in some profiles.

Reply:
Yeah, I'll open another PR for that

Reply:
Don't forget this please :)

/lgtm

Reply:
Opened here: https://github.com/kubernetes/kubernetes/pull/99807